fixham/internal/operator/git.go

```diff
@@ -0,0 +1,46 @@
+package operator
+
+import (
+	"os"
+
+	"github.com/projectcalico/calico/fixham/internal/command"
+)
+
+const (
+	operatorRepo = "git@github.com:tigera/operator.git"
+)
+
+func operatorDir(repoRootDir string) string {
+	return repoRootDir + "/operator"
```

Comment:
We shouldn't be cloning the operator (or any other intermediate files) into the root of the projectcalico/calico repo.

I think the best thing to do is to use a temporary directory generated for each invocation. Each directory should be unique and include the hashrelease name / version so that there is no chance of accidentally using the wrong clone if, for example, you run multiple hashreleases back-to-back on different branches.